mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -204,6 +204,15 @@ def ROCDL_ReadlaneOp : ROCDL_IntrOp<"readlane", [], [0], [AllTypesMatch<["res",
    }];
 }
 
+// The LLVM intrinsic function name is rather mouthful,
+// so here we opt to use a shorter rocdl name.
+def ROCDL_WaveIdOp :  LLVM_IntrOpBase<ROCDL_Dialect, "wave_id",
```

Comment:
I object to the name change - this is a rather low-level dialect, so we should try to mirror the LLVM-side names

Reply:
I was not sure about this. If generally ROCDL ops should map exactly to LLVM intrinsics, then I am okay with using the long name here. I will make the change.